InventoryCatalogFrontendUi/Model/QtyLeftChecker.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalogFrontendUi\Model;
+
+use Magento\InventoryConfigurationApi\Api\Data\StockItemConfigurationInterface;
+
+/**
+ * Check if it is necessary to show qty left.
+ */
+class QtyLeftChecker
```

Comment:
Rename to `isSalableQtyAvailableForDisplaying `

Reply:
@sidolov did you mean change class name to isSalableQtyAvailableForDisplaying?
Maybe from capital letter IsSalableQtyAvailableForDisplaying?

Reply:
Yes, I meant the first capital letter - `IsSalableQtyAvailableForDisplaying `

Reply:
Fixed